tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoHAAdmin.java

```diff
@@ -131,6 +131,9 @@ public void runCommand(String[] args) throws Exception {
     } else if (hostName == null && port == null) {
       tajoClient = new TajoClientImpl(ServiceTrackerFactory.get(tajoConf));
     }
+    if (!tajoClient.isConnected()) {
```

Comment:
I think that this class also does not require TajoClient connection. Removing TajoClient would be a better solution to fix 'unused field' warning.